benchmarks/Surrogates/cantilever.jmd

```diff
@@ -0,0 +1,160 @@
+---
+title: Cantilever Beam Function
+author: Mridul Jain, Chris Rackauckas
+---
+
+The Cantilever Beam function is defined as:
```

Comment:
What is this function exactly? I understand its related to Cantilever Beam but what does it compute?

Reply:
To the best of my knowledge, engineers need numerical solvers for computational analysis and simulations which are precisely performed on cantilever beams.